lbr.py

```diff
@@ -720,15 +720,36 @@ def print_glob_hist(hist, name, weighted=False, Threshold=0.01):
   print('%s histogram summary: %s' % (name, hist_fmt(d)))
   return d['total']
 
-def print_hist(hist_t, Threshold=0.01):
+def print_hist(hist_t, Threshold=0.01, hitcounts=None):
+  def tc_mean():
+    hex_ipc = '0%x' % loop_ipc
+    hotness = int(C.str2list(C.exe_one_line('%s' % C.grep(hex_ipc, hitcounts)))[0])
+    before = after = 0
+    res = C.exe_one_line('%s' % C.grep(hex_ipc, hitcounts, '-B1'))
+    l = C.str2list(res)
+    if hex_ipc in l[1]: return None  # no inst before loop
+    # JCC before loop is not considered a special case
```

Comment:
update comment to *directly* reflect next line of code JMP_RET. Now it seem to implicit state things a 3rd user may not follow